javascript/modules/client/src/internal/client-core.ts

```diff
@@ -103,13 +117,29 @@ export abstract class ClientCore implements IClientCore {
     return this._daoFactoryAddress;
   }
 
+  get ipfs() {
+    return this._ipfs;
```

Comment:
I'd return a usable one, not just all.

```suggestion
    if (!this._ipfs[this._ipfsIdx]) throw new Error("No IPFS endpoints available");
    return this._ipfs[this._ipfsIdx];
```